frontend/src/pages/spotnet/dashboard/withdraw-all/WithdrawAll.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import BalanceCards from 'components/BalanceCards';
+import { ReactComponent as ETH } from '../../../../assets/icons/ethereum.svg';
+import { ReactComponent as USDC } from '../../../../assets/icons/borrow_usdc.svg';
+import { ReactComponent as STRK } from '../../../../assets/icons/strk.svg';
+import './withdraw_all.css';
+import { useWalletStore } from 'stores/useWalletStore';
+import Button from 'components/ui/Button/Button';
+import { ReactComponent as AlertHexagon } from 'assets/icons/alert_hexagon.svg';
+import { axiosInstance } from 'utils/axios';
+
+const WithdrawAll = () => {
+  const { walletId } = useWalletStore();
+  const [balances, setBalances] = useState([
+    { icon: <ETH />, title: 'ETH', balance: '0.00' },
+    { icon: <USDC />, title: 'USDC', balance: '0.00' },
+    { icon: <STRK />, title: 'STRK', balance: '0.00' },
+  ]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const handleWithdrawAll = async () => {
+    setLoading(true);
+    setError(null);
+
+    try {
+      const response = await axiosInstance.get(`/api/withdraw-all?wallet_id=${walletId}`);
+      console.log(response);
+      if (response.status === 200) {
+        console.log('Withdraw All operation completed successfully.');
+      } else {
+        console.log('Failed to complete the Withdraw All operation.');
+      }
+    } catch (error) {
+      console.error('Error during Withdraw All operation:', error);
+      setError('Something went wrong. Please try again.');
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <div className="withdrawall-wrapper">
+      <div className="withdrawall-container">
+        <BalanceCards balances={balances} setBalances={setBalances} walletId={walletId} />
```

Comment:
Please make sure that balance cards styles replicate figma. Currently background, font colors, weights, padding between elements do not match figma. Also, you can see that there are no more dots (pagination on mobile) below cards, please remove it (component can be removed at all)

Reply:
Cards should not be "cut" like this, but partially visible juist like in a design, also space between cards should be accroding to figma. Also space from the left of first card is too big (mobile)
![image](https://github.com/user-attachments/assets/e46187fb-2a2a-463d-96a8-d196efc4b648)

Reply:
Updated!

Reply:
1) mobile padding should be smaller, 8px, instead of 20px, so the 3rd card part will be visible as in design
2) Also balance title on mobile should have the same color as in figma (just like desktop)
3) desktop font-weight of balance amount is not equal to 600 like in figma (however mobile is fine, make the same as ther)